tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.PublicApiHasNotChanged_netcoreapp.verified.txt

```diff
@@ -0,0 +1,570 @@
+[assembly: System.Reflection.AssemblyMetadata("RepositoryUrl", "https://github.com/DataDog/dd-trace-dotnet.git")]
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Azure.Functions
```

Comment:
You can delete this file. This is a remnant of pre-rebase days. Will also make your add/delete ratio look more favourable 😉

Reply:
Thanks! Fixed in b5c3f7ed